tests/test_cases/test_vhdl_libraries_multiple/test_abcde.py

```diff
@@ -0,0 +1,11 @@
+import cocotb
+
+
+@cocotb.test()
+async def test(dut):
+    # Toggling an input should trigger the simulator to print a message
+    # similar to:
+    #
+    #   e.vhdl:9:5:@0ms:(report note): :a(structural):b@b(structural):c@c(structural):d@d(structural):e@e(structural)
+    #
+    dut.x <= False
```

Comment:
If you aren't testing the output, there isn't a reason to trigger it.

```suggestion
    pass
```